apps/hash-integration-worker/package.json

```diff
@@ -7,7 +7,8 @@
   "exports": {
     ".": "./src/main.ts",
     "./workflows": "./src/workflows.ts",
-    "./activities": "./src/activities.ts"
+    "./activities": "./src/activities.ts",
+    "./graph": "./src/graph.ts"
   },
   "typesVersions": {
```

Comment:
You can remove these two objects entirely – they are to declare what exports are available from this package for other apps/packages to use, but this is not designed to be imported by other packages. Apologies for not spotting this sooner.